### PR TITLE
use MIDDLEWARE from 1.10 on

### PR DESCRIPTION
**Pull request by manelclos**

### CODE REVIEW

**Comment by bmihelac** [comment]
Looks good to me. The only thing I am not sure if we should drop supporting old Django versions: https://github.com/django-import-export/django-import-export/issues/681

File: tests/settings.py [hunk 1]
```diff
@@ -33,6 +36,16 @@
     'django.middleware.clickjacking.XFrameOptionsMiddleware',
 )
 
+if VERSION < (1, 10, 0):
+    MIDDLEWARE_CLASSES = (
```
Comment (mgrdcm) [inline tests/settings.py]: Could this just be `MIDDLEWARE_CLASSES = MIDDLEWARE`?

Comment (manelclos) [reply tests/settings.py]: yes! will update. Working on other more complicated django-2.0 support

**Comment by bmihelac** [comment]
this is resolved with 3ea43166150b5bef2f2d8422c301a865f8413db1

**Pull request closed without merging**